my-code/movie-search/src/Containers/SearchBar/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchMovies } from '../../Actions';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { searchTerm: '' };
+  }
+
+  onInputChange(searchTerm) {
+    this.setState({ searchTerm });
+    this.props.fetchMovies(searchTerm);
+  }
+
+  render() {
+    return (
+      <div className="SearchBar">
+        <input
+          placeholder="Search movies..."
+          value={this.state.searchTerm}
+          onChange={event => this.onInputChange(event.target.value)} />
```

Comment:
You could just pass the function reference and get event.target.value there instead of creating a new anonymous function on each render

Reply:
Noted!